aws/aws_test.go

```diff
@@ -11,9 +11,9 @@ import (
 )
 
 func TestWriteToFile(t *testing.T) {
```

Comment:
I would move the expiration test to a new test case. A test case should ideally test one thing.
The `TestWriteToFile` test case should only check that writing credentials to file works. We should have *another* test case which checks that expiration logic is handled properly. Doing so will make it easier to figure out what is wrong upon test failure.

Reply:
How about it?